Calculator/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using CalculatorLibrary;
+using Newtonsoft.Json;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool endApp = false;
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
+
+            CalculatorProgram calculatorProgram = new CalculatorProgram();
+
+            while (!endApp)
+            {
+                // Declare variables and set to empty.
+                string numInput1 = "";
+                string numInput2 = "";
+                double result = 0;
+                string histResult;
+
+                Console.WriteLine("What would you like to do?");
+                Console.WriteLine("\tv - View History");
+                Console.WriteLine("\to - Calculator");
+                Console.WriteLine("\te - Exit");
+                Console.Write("Your option?");
+                string opt = Console.ReadLine();
+                if (opt == "v")
+                {
+                    // Calling the History Method
+                    calculatorProgram.EndHistory();
+                    Console.Write("Do you want to clear the history? Press 'y' to clear the history:");
+                    string clearHistory = Console.ReadLine();
+                    if (clearHistory == "y")
+                    {
+                        calculatorProgram.ClearHistory();
+                    }
```

Comment:
Can be separated into it's own method, and comment removed by making the method name descriptive.